Rakefile

```diff
@@ -38,10 +38,12 @@ task :dist do
   #env['asciidoctor'].write_to "build/asciidoctor.js#{compress ? '.gz' : nil}"
   asciidoctor = env['asciidoctor']
   # NOTE hack to make version compliant with semver
-  asciidoctor_src = env['asciidoctor/core_ext/match_data'].source + env['asciidoctor'].source
+  core_ext_files = ['dir', 'error', 'file', 'match_data', 'opal_ext']
```

Comment:
Why not just require `asciidoctor/opal_ext` and declare requires in that file instead of itemizing each one in the build?